back/src/main/java/com/ubikloadpack/jmeter/ulp/observability/log/SampleLog.java

```diff
@@ -161,6 +173,7 @@ public SampleLog(
 		this.maxTotal = maxTotal;
 		this.avgTotal = avgTotal;
 		this.errorTotal = errorTotal;
+		this.topErrors = topErrors == null ? new ErrorsMap() : topErrors;
```

Comment:
because we shouldn't pass the topErrors if the sample doesn't represent the total_label. The SampleLog is created for all samples including the totalLabel. So in the case of other samples we don't pass the topErrors. This is done in https://github.com/ubikingenierie/ulp-observability-plugin/blob/88_RI_dev/back/src/main/java/com/ubikloadpack/jmeter/ulp/observability/registry/MicrometerRegistry.java#L245

Reply:
In this case it's better to use Optional<ErrorsMap>

Reply:
ok